src/jobs/heartbeat.ts

```diff
@@ -0,0 +1,29 @@
+import { v1 } from '@datadog/datadog-api-client';
+import * as Sentry from '@sentry/node';
+import moment from 'moment-timezone';
+
+import { DATADOG_API_INSTANCE } from '@/config';
+
+export async function callDatadog(timestamp: number): Promise<void> {
+  const params: v1.EventCreateRequest = {
+    title: 'Infra Hub Update',
+    text: 'Infra Hub is up',
+    alertType: 'error',
```

Comment:
What does an `error` `alertType` imply? I didn't find much info after glancing at their docs.

Reply:
I think it's just the type of event that is sent